src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$main-accent: #060b35;
+$secondary-accent: #616070;
+$blue-accent: #00acdc;
+$white-color: #fff;
```

Comment:
#checklist.md
BAD example:
```scss
$main: #060b35;
$secondary: #616070;
$blue: #00acdc;
```

GOOD example:
```scss
$color-main-accent: #060b35;
$color-secondary: #616070;
$color-accent-blue: #00acdc;
```

Reply:
➕ , not fixed